src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorOperation.cs

```diff
@@ -2103,6 +2103,26 @@ public static void Invoke(ReadOnlySpan<T> x, ReadOnlySpan<T> y, Span<T> destinat
             }
         }
 
+        public readonly struct SumOfSquaredMagnitudeDifferences<T>
```

Comment:
nit: I'm not a huge fan of `MagnitudeDifferences` as it reads to me like `abs(x) - abs(y)` instead of `abs(x - y)`

I'm more used to "sum of absolute differences" being used to refer to `abs(x - y)`; which is notably what both x64 and Arm64 use in their SIMD instruction support as well.

Not a big deal either way, just a case where I think "absolute difference" vs "magnitude difference" reads a little different despite intending to mean the "same thing"